aspnetcore/fundamentals/host/web-host.md

```diff
@@ -51,6 +51,41 @@ public class Program
 * When running behind IIS, enables [IIS integration](xref:host-and-deploy/iis/index). Configures the base path and port the server listens on when using the [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module). The module creates a reverse proxy between IIS and Kestrel. Also configures the app to [capture startup errors](#capture-startup-errors). For the IIS default options, see [the IIS options section of Host ASP.NET Core on Windows with IIS](xref:host-and-deploy/iis/index#iis-options).
```

Comment:
Line 44: Loads app configuration from ...
This could also use a separate entry for host configuration:
- Env vars
- command line params.